.travis.yml

```diff
@@ -35,6 +35,8 @@ stages:
     if: branch = master AND type = push
   - name: e2e
     if: type = cron || type = pull_request
+  - name: Release Libraries
+    if: (branch = develop || branch = master) AND type = push
```

Comment:
why do we need develop or master ?

Reply:
u already have 
branches:
  only:
    - master
    - develop

Reply:
Ok, so we only need to have `type = push` so we are going to publish libraries only when the develop or master is buld